client/blog-app/src/App.js

```diff
@@ -1,7 +1,92 @@
 import "./App.css";
+import Feed from "./components/Feed";
+import { Routes, Route, useNavigate, Switch } from "react-router-dom";
+import CreateNewPost from "./components/CreateNewPost";
+
+const mock = [
```

Comment:
I think its better to move it to a different file but it doesn't really matter now :)